airflow/www/views.py

```diff
@@ -246,10 +266,20 @@ def get_int_arg(value, default=0):
                     DagModel.owners.ilike('%' + arg_search_query + '%')
                 )
 
+            if arg_tags_filter:
+                dags_query = dags_query.filter(DagModel.tags.any(DagTag.name.in_(arg_tags_filter)))
+
             if 'all_dags' not in filter_dag_ids:
                 dags_query = dags_query.filter(DagModel.dag_id.in_(filter_dag_ids))
 
             dags = dags_query.order_by(DagModel.dag_id).offset(start).limit(dags_per_page).all()
+            tags = []
+
+            for name, in session.query(DagTag.name).distinct(DagTag.name).all():
```

Comment:
Is this comma a typo? This query only returns a single column right?

Reply:
It will select only one column, but it will return a list of tuple, so the comma is in order to unpack the first item.